src/app/shared-component/status-drawer.component.html

```diff
@@ -1,20 +1,17 @@
 						
 <div *ngIf="workItem" class="status">
-	<span 
-		class="label" 
-		(click)="onDrawerToggle()"
-		[ngClass]="{'label-warning': workItem.statusCode==0, 
+	<span class="label" (click)="onDrawerToggle()" id="statusDrawer_workItemStatus" [ngClass]="{'label-warning': workItem.statusCode==0, 
 					'label-primary': workItem.statusCode==1, 
 					'label-success': workItem.statusCode==2,
 					'label-info': workItem.statusCode==3 }">
 		
 		{{workItem.fields['system.state'] ? workItem.fields['system.state'] : "Status"}}
 	</span>
 	<ul *ngIf="show" class="drawer">
-		<li (click)="changeStatus(0,'new')">New</li>
-		<li (click)="changeStatus(1,'in progress')">In Progress</li>
-		<li (click)="changeStatus(2,'resolved')">Resolved</li>
-		<li (click)="changeStatus(3,'closed')">Closed</li>
+		<li id="statusDrawer_ListItemStatusNew" (click)="changeStatus(0,'new')">New</li>
```

Comment:
This is a sub component and used multiple number of times in the DOM. Putting a static ID on it will lead to ID redundancy.